### DISCUSSION TITLE
A small mistake in documentation

### DISCUSSION THREAD

**Discussion by ZaeemKhaliq** [HaylLtd/cordova-plugin-background-geolocation#61] [General]
Please check in documentation on the API page under the Configure Options section, for the property **distanceFilter** of the configure object, the provider value written in documentation is both DIS and RAW. Whereas if I am not wrong the **distanceFilter** is supposed to be used with only DISTANCE_FILTER_PROVIDER??

![image](https://user-images.githubusercontent.com/57555591/147450741-75259814-9062-4164-8ec4-9b06601d06e8.png)

**Comment by HarelM** [comment]
Thanks for reaching out! I'm not 100% sure, but I believe the distance filter is used in RAW provider by filtering the events sent to the client. i.e. the filter is in this plugin's code and not the one from the OS.
Feel free to poke around the code and correct me if I'm wrong. If I am, please send a PR :-)

**Reply by ZaeemKhaliq** [reply]
Oh, yes I just tested the **distanceFilter** with RAW_PROVIDER and it triggers location update event after travelling for the value in meters specified in **distanceFilter.**
Actually, I was confused as to what actually then is the purpose of RAW_PROVIDER if it uses **distanceFilter** too. Logically, only DISTANCE_FILTER_PROVIDER should use **distanceFilter**. 
Never mind though.
